about.html

```diff
@@ -0,0 +1,349 @@
+                <li class="for-desktop">
+                  <a
+                    href="https://wwww.twitter.com"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    <img
+                      class="nav-bar-icon-logo"
+                      src="resources/icons/twitter-logo.png"
+                      alt="Twitter Logo"
+                    />
+                  </a>
+                </li>
+                <li class="for-desktop">
+                  <button
+                    type="button"
+                    class="nav-bar-icon-logo-li transition font"
+                  >
+                    English
+                  </button>
+                </li>
+                <li class="for-desktop">
+                  <button
+                    type="button"
+                    class="nav-bar-icon-logo-li transition font"
+                  >
+                    My Page
+                  </button>
+                </li>
+                <li class="for-desktop">
+                  <button
+                    type="button"
+                    class="nav-bar-icon-logo-li transition font"
+                  >
+                    Logout
+                  </button>
+                </li>
+              </ul>
+              <div class="nav-bar-logo-div">
+                <section class="logo">
+                  <a class="header-logo transition font" href="index.html"
+                    >©️ComedyBurst</a
+                  >
+                </section>
+                <ul>
+                  <li>
+                    <a
+                      class="bottom-header-ul-li-text transition font"
+                      href="index.html"
+                      >Home</a
+                    >
+                  </li>
+                  <li>
+                    <a class="bottom-header-ul-li-text transition font" href="#"
+                      >Event</a
+                    >
+                  </li>
+                  <li>
+                    <a class="bottom-header-ul-li-text transition font" href="#"
+                      >Join</a
+                    >
+                  </li>
+                  <li>
+                    <a class="bottom-header-ul-li-text transition font" href="#"
+                      >Sponsor</a
+                    >
+                  </li>
+                  <li>
+                    <a class="bottom-header-ul-li-text transition font" href="#"
+                      >News</a
+                    >
+                  </li>
+                </ul>
+                <button type="button" class="campaign transition font">
+                  ComedyBurst Initiative
+                </button>
+              </div>
+            </section>
+          </div>
+        </div>
+        <div class="about-page-header-body for-phone">
+          <section class="about-header-text-div">
+            <h1 class="about-header-text-head font">"Hey! Comic lovers"</h2>
```

Comment:
- [ ] Kindly note that the `h1` tag should be used only once on each page, kindly use another `hx`.

Reply:
Done 👍 @Ofarouq310